src/uploader/views/submission_views.py

```diff
@@ -18,6 +19,7 @@ class SubmissionCreateView(ConfiguredLoginViewMixin, CreateView):
     template_name = "uploader/upload_form.html"
     success_url = reverse_lazy("index")
     update_name = "update-from-youtube"
+    submit_action = download_song
```

Comment:
is this tested?

Reply:
![grafik](https://user-images.githubusercontent.com/52469686/227310638-5c4d2179-c3ae-486c-83d9-f8f2f47b8890.png)
I dont know this existed

Reply:
ah well, there is a django thing simmilar to this, but this is just a var to reuse the view